### PR TITLE
[misc] Fix python wheel versioning.

### PR DESCRIPTION
**Pull request by ailzhang**
Related issue = #

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,6 +35,10 @@
 ]
 
 project_name = os.getenv('PROJECT_NAME', 'taichi')
+TI_VERSION_MAJOR = 0
+TI_VERSION_MINOR = 8
+TI_VERSION_PATCH = 5
+version = f'{TI_VERSION_MAJOR}.{TI_VERSION_MINOR}.{TI_VERSION_PATCH}'
```
Comment (rexwangcc) [inline setup.py:41]: FTR: this is the source of truth of the version info.

**Review by rexwangcc — APPROVED** [review]
👍 LGTM! Thanks!!

**Comment by strongoier** [comment]
It seems that this change reverts what was done in #3354. Which way should we take? We need to avoid modify this back-and-forth IMO :-)

**Comment by frostming** [comment]
How are these values to be updated? Maybe we can get them from git tags automatically. I am happy to provide an implementation.

**Comment by strongoier** [comment]
> How are these values to be updated? Maybe we can get them from git tags automatically. I am happy to provide an implementation.

@ailzhang @rexwangcc WDYT? In this case we no longer need to manually set versions.

**Comment by ailzhang** [comment]
Hey @frostming @strongoier, sorry for the late reply! @frostming would you mind describing your "fetching from git tags automatically" idea in detail in a new issue? 
My main concern is if someone download our zipped source code (it is a folder without `.git/` in this case), then will they be able to get the version information? As long as this work I'll be super happy to see this process being automatic! :D

**Pull request merged**